NOTICES.md

```diff
@@ -0,0 +1,80 @@
+# Implemented notices
```

Comment:
We may want to consider referring to `Notices` in more casual text that will be read by agencies as "rules" - I think that's more common in the context of validation. The fact that they are called "notices" are more of an implementation detail.